tutorials/misc/plot_report.py

```diff
@@ -203,6 +203,30 @@
 
 report.save('report_custom.html', overwrite=True)
 
+###############################################################################
+# Adding a stc plot to a report
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Now we see how :class:`~mne.Report` handles :class:`~mne.SourceEstimate`
+# data. The following will produce a stc plot with vertex time courses. In
+# this scenario, we also demonstrate how to use the ``screenshot`` function
+# to save the figs in a slider.
+
+fname_stc = os.path.join(path, 'MEG', 'sample', 'sample_audvis-meg')
```

Comment:
If you look at the output you actually have to scroll quite far in the report to see what these lines actually accomplish:

https://24767-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/misc/plot_report.html#adding-a-stc-plot-to-a-report

This can be fixed by making a new report like is done in the earlier sections:
```suggestion
report = mne.Report(verbose=True)
fname_stc = os.path.join(path, 'MEG', 'sample', 'sample_audvis-meg')
```
The section immediately precedes this, "Adding custom plots to a report", has the same problem, if you could fix that, too, it would be great